src/Lean/Elab/Command.lean

```diff
@@ -503,6 +504,45 @@ def expandDeclId (declId : Syntax) (modifiers : Modifiers) : CommandElabM Expand
 
 end Elab.Command
 
+open Elab Command MonadRecDepth
+
+/--
+Lift an action in `CommandElabM` into `CoreM`, updating the traces and the environment.
+This does not preserve things like `open` and `namespace` declarations.
```

Comment:
What does "preserve" mean in this context?

Perhaps this means the same thing?
```suggestion
Commands that modify the processing of subsequent commands, such as `open` and `namespace`
declarations, only have an effect for the remainder of the `CommandElabM` computation passed here,
and do not affect subsequent commands.
```